apps/browser/components/Chat.tsx

```diff
@@ -120,8 +94,14 @@ const Chat: React.FC<ChatProps> = ({
           <TextField
             type="text"
             value={message}
-            onChange={handleMessageChange}
-            onKeyDown={handleKeyPress}
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              setMessage(event.target.value)
+            }}
+            onKeyDown={(event: React.KeyboardEvent) => {
+              if (event.key === "Enter" && !isStarting && !isRunning) {
```

Comment:
Should `!isStarting && !isRunning` be part of `handleGoalSubmit`?

Reply:
Potentially yes, would need to test.